spec/presenters/sufia/trophy_presenter_spec.rb

```diff
@@ -16,7 +16,7 @@
   end
 
   let(:presenter) { described_class.new(solr_document) }
-  let(:solr_document) { SolrDocument.new(id: '123456', has_model_ssim: 'GenericWork') }
+  let(:solr_document) { SolrDocument.new(id: '123456', has_model_ssim: 'GenericWork', title_tesim: 'A Title') }
```

Comment:
In Behavior. It does `Array.wrap` with title, so the expectation seems to be either string or array.